api/src/main/resources/org/openmrs/liquibase/snapshots/core-data/liquibase-core-data-2.7.x.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+
+    This Source Code Form is subject to the terms of the Mozilla Public License,
+    v. 2.0. If a copy of the MPL was not distributed with this file, You can
+    obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+    the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+
+    Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
```

Comment:
You do not have to touch this file at all because we are not going to insert any core data.

Reply:
Kindly note that this was test data and wasn't meant to be committed.